sklearn/ensemble/tests/test_gradient_boosting.py

```diff
@@ -743,6 +804,36 @@ def test_warm_start_clear(Cls):
     assert_array_almost_equal(est_2.predict(X), est.predict(X))
 
 
+@pytest.mark.parametrize("GradientBoosting", GRADIENT_BOOSTING_ESTIMATORS)
+def test_warm_start_state_oob_scores(GradientBoosting):
+    """Check that the states of the OOB scores when used with `warm_start`."""
```

Comment:
This sentence somehow got garbled. Unfortunately I can't work out what it should be, otherwise I'd make a suggested fix.

Reply:
Check that the states of the OOB scores are cleared depending on `warm_start` parameter.

(Indeed, I forgot some of the words ;))